parsing/parsetree.mli

```diff
@@ -468,6 +476,7 @@ and expression_desc =
          *)
   | Pexp_send of expression * label loc  (** [E # m] *)
   | Pexp_new of Longident.t loc  (** [new M.c] *)
+    (* jra: should this be renamed? Both mutable vars and instance vars use this syntax *)
```

Comment:
Sounds like a good idea.  Maybe just `Pexp_setvar`?  Probably the comment on this case should note the two things it works on.